util/cron/test-perf.hpe-apollo-hdr.arkouda.playground.bash

```diff
@@ -54,8 +54,8 @@ GITHUB_BRANCH=main
 SHORT_NAME=playground
 START_DATE=09/03/25
```

Comment:
make sure to update SHORT_NAME and SHORT_DATE

Reply:
Done! I made it obvious that I didn't read the manual. Sorry about that.